### PR TITLE
Be less tolerant with the possibility to have a mask

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: buildtools/check-example [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+import tempfile
+
+
+def main():
```
Comment (arnaud-morvan) [inline buildtools/check-example:9]: Would be helpful to have a short description of what does this script do and an example usage.
A idea about how to create a mask, some example using CLI ?

File: buildtools/check-example [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    diff = current.replace(".html.png", "-diff.png")
+
+    if os.path.exists(mask):
+        print("Apply mask")
+        old_current = current
+        old_ref = ref
+        current = tempfile.NamedTemporaryFile(suffix="-current.png").name
+        ref = tempfile.NamedTemporaryFile(suffix="-ref.png").name
+        subprocess.check_call(
+            ["gm", "composite", "-compose", "Multiply", old_current, main, current]
```
Comment (arnaud-morvan) [inline buildtools/check-example:22]: Shouldn't it be "mask" instead of "main" here ?
Same on next line.

**Review by arnaud-morvan — APPROVED** [review]
Seems a good move.

File: buildtools/check-example (hunk 2 above)
Comment (sbrunner) [reply buildtools/check-example:22]: +1 Thanks :-)

File: buildtools/check-example (hunk 1 above)
Comment (sbrunner) [reply buildtools/check-example:9]: Just done :-)

File: buildtools/check-example [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import sys
+import tempfile
+
+
+def main():
+    """
+    Compare similarity of tow images, we can use an optional mash file.
+    On error a diff image is generated.
+
+    Usage: check-examle <generated image> <referanc image>
```
Comment (arnaud-morvan) [inline buildtools/check-example:14]:
```suggestion
    Usage: check-example <generated image> <reference image>
```

Comment (sbrunner) [reply buildtools/check-example:14]: Thanks :-)

**Pull request merged**